ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.txt

```diff
@@ -27,7 +27,7 @@ hideall "Liquid Gaol"
 71.9 "Protean Wave 2" sync /:Living Liquid:4823:/
 74.9 "Sluice" sync /:Liquid Rage:49B1:/
 75.0 "Protean Wave 3" sync /:Living Liquid:4823:/
-81.1 "Splash" sync /:Living Liquid:49B2:/ window 3,0
+81.1 "Splash x6" sync /:Living Liquid:49B2:/ window 3,0
```

Comment:
I know this is not your original line, but rather than having a window with zero lookahead here, could you remove the sync entirely and add a duration?

Same thing below with the other splashes and the apocalyptic rays.

Reply:
Done. I think Apocalyptic Rays and Hawk Blast behave similarly as in the player moves out of the targetted (unmarked?) AoE. I haven't read over much of this part of the fight nor am I familiar with the original fights.